src/config/config.go

```diff
@@ -64,11 +64,11 @@ func ValidDateConfiguration(config *Configuration) {
 			fmt.Println("Please enter a valid pathway to the static file")
 		}
 	} else if fileInputL != "live" && fileInputL != "static" {
-		fmt.Println("Please enter either live or static as a dataOption in config.json")
+		fmt.Println("Please enter either live or static as a dataOption in default-config.json")
```

Comment:
Should have caught this earlier - but we should probs return an error and not continue program execution if the file does not have valid content.

Reply:
oh, yeah, strong agree; bail early if in doubt

Reply:
^Jolene is working on the next pull request for all config related stuff; I passe this comment to her